.github/workflows/build-and-test.yml

```diff
@@ -399,13 +399,13 @@ jobs:
       - name: Installation test
         run: |
           $ErrorActionPreference = 'Stop'
-          Set-PSDebug -Trace 1
+          # Set-PSDebug -Trace 1
```

Comment:
I don't know from the top of my head how to do it, but, instead of commenting out this line it may be worth making it use an input for the workflow, so it has a desired default for the automatic runs but you can provide a different value when running it manually.

Reply:
Thanks.  I just commented it out as a temporary change to decrease the noise during my testing.  We can use https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs to change this for manual runs, if necessary.